monitor/services/src/main/java/org/datacleaner/monitor/server/job/DataCleanerJobContext.java

```diff
@@ -39,6 +40,9 @@
 
     AnalysisJob getAnalysisJob(Map<String, String> variableOverrides, Map<String, String> overrideProperties);
 
+    AnalysisJob getAnalysisJob(Map<String, String> variableOverrides, Map<String, String> overrideProperties,
+            Datastore datastore);
```

Comment:
This is an interface. `final` doesn't make any sense here? Our CheckStyle rules would even fail a final because it is redundant.